pkgs/misc/tmux-plugins/default.nix

```diff
@@ -89,12 +89,12 @@ in rec {
 
   cpu = mkDerivation {
     pluginName = "cpu";
-    version = "unstable-2020-04-05";
+    version = "unstable-2020-07-25";
```

Comment:
Please use these versions in the commit message.

Reply:
Fixed